go/libraries/doltcore/sqle/tables.go

```diff
@@ -726,6 +729,87 @@ func (t *AlterableDoltTable) DropForeignKey(ctx *sql.Context, fkName string) err
 	return t.updateFromRoot(ctx, newRoot)
 }
 
+// GetForeignKeys implements sql.ForeignKeyTable
+func (t *AlterableDoltTable) GetForeignKeys(ctx *sql.Context) ([]sql.ForeignKeyConstraint, error) {
+	root, err := t.db.GetRoot(ctx)
+	if err != nil {
+		return nil, err
+	}
+	fkc, err := root.GetForeignKeyCollection(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	fks, _ := fkc.KeysForTable(t.name)
```

Comment:
This should be `fks, err := fkc.KeysForDisplay(ctx, t.name, root)`, as `toForeignKeyConstraint` is essentially duplicating that function

Reply:
Much better, thanks.